server/Services/LoginService.js

```diff
@@ -1,37 +1,65 @@
+const TYPES = { "GroceryStores": "Grocery Store", "FoodBank": "Food Bank", "Drivers": "Driver" };
```

Comment:
can we make food bank key consistent with others (as plural)?

Reply:
Yeah I'll rename the collection and update the code.